lib/rubocop/result_cache.rb

```diff
@@ -128,7 +128,8 @@ def rubocop_checksum
         begin
           lib_root = File.join(File.dirname(__FILE__), '..')
           bin_root = File.join(lib_root, '..', 'bin')
-          source = Find.find(lib_root, bin_root).sort.map do |path|
+          source_files = $LOADED_FEATURES + Find.find(lib_root, bin_root).to_a
```

Comment:
At the very least we should have a comment here explaining what this line of code does. Even I had to think a bit about it when I saw it.